continuous_integration/install.sh

```diff
@@ -76,7 +76,7 @@ if [ -z "$NUMPY_VERSION" ]; then
 fi
 
 if [[ "$COVERAGE" == "true" ]]; then
-    pip install coverage coveralls
+    pip install coverage coveralls memory_profiler
```

Comment:
What I am just trying to say, don't add anything related to memory_profiler to this PR. This is enough work to get it merged already.